docs/how-to/prevent-slashing/detect-slashing.md

```diff
@@ -0,0 +1,67 @@
+---
+description: Shut down Teku if a validator is slashed.
+sidebar_position: 3
+---
+
+# Stop Teku when a validator is slashed
+
+Validator slashing detection monitors slashing events and shuts down Teku when any owned validator
+is slashed.
+This can help prevent further slashing offenses.
+
+When a validator is slashed, the Teku validator client shuts down with exit code `2`.
+When this happens, you should not restart Teku by default because validators will likely continue to
+be slashed.
+We recommend restarting Teku with [doppelganger detection](detect-doppelgangers.md) enabled.
+
+:::warning
+
+This feature is imperfect and might fail to detect slashing events rapidly.
+Use it as a last resort option that might prevent validators from being slashed.
+
+:::
+
+## Enable validator slashing detection
+
+Enable validator slashing detection by setting the `--shut-down-when-validator-slashed-enabled`
+option to `true`.
+
```

Comment:
```suggestion

When a validator is slashed, you must remove it from the owned validators before restarting Teku with validator slashing detection enabled.
Otherwise, the validator will still be detected as slashed and Teku will shut down again.
The duties performed by a slashed validator are ignored, and it's queued for exit after being slashed.
If you wish to keep it running, you should disable validator slashing detection before restarting Teku.

```

I think this text fits better in this section because it's more of a recommended practice related to enabling/disabling slashing detection vs. a side effect of shutting down Teku.